mne/report.py

```diff
@@ -82,7 +82,11 @@ def _scale_mpl_figure(fig, scale):
         sfactor = -((1. / scale) ** 2)
     for text in fig.findobj(mpl.text.Text):
         fs = text.get_fontsize()
-        text.set_fontsize(fs + sfactor)
+        try:
+            text.set_fontsize(fs + sfactor)
+        except RuntimeError:
```

Comment:
does this actually fix the inverted fonts? I thought you saw it because there was no error